### PR TITLE
add rag exercise nb

### PR DESCRIPTION
**Pull request by kylesteckler**

### CODE REVIEW

File: notebooks/vertex_genai/labs/retrieval_augmented_generation.ipynb [hunk 1]
```diff
@@ -0,0 +1,551 @@
+{
```
Comment (BenoitDherin) [inline notebooks/vertex_genai/labs/retrieval_augmented_generation.ipynb:1]: <ul><li>Add a markdown cell with s&nbsp;<code>### Exercise</code>&nbsp;title, and a description of the exercise.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/419/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (BenoitDherin) [inline notebooks/vertex_genai/labs/retrieval_augmented_generation.ipynb:1]: <ul><li>Add a markdown cell with s&nbsp;<code>### Exercise</code>&nbsp;title, and a description of the exercise.</li><li>You can ask to implement the body of the function and  describe its signature and the details of what it is suppose to do in the markdown cell above.</li><li>"prompt = #" may trigger pre-commit issues. Please use "prompt = None # etc."</li></ul><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/419/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (BenoitDherin) [inline notebooks/vertex_genai/labs/retrieval_augmented_generation.ipynb:1]: <ul><li>Add a markdown cell with s&nbsp;<code>### Exercise</code>&nbsp;title, and a description of the exercise.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/419/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (BenoitDherin) [inline notebooks/vertex_genai/labs/retrieval_augmented_generation.ipynb:1]: <ul><li>Add a markdown cell with s&nbsp;<code>### Exercise</code>&nbsp;title, and a description of the exercise.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/419/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (kylesteckler) [reply notebooks/vertex_genai/labs/retrieval_augmented_generation.ipynb:1]: <p>done</p>

Comment (kylesteckler) [reply notebooks/vertex_genai/labs/retrieval_augmented_generation.ipynb:1]: <p>done</p>

**Review by BenoitDherin — APPROVED** [review]
LGTM

**Pull request merged**